### DISCUSSION TITLE
Show off your production ready MAUI apps

### DISCUSSION THREAD

**Discussion by BlagoCuljak** [dotnet/maui#9599] [Show and tell]
Here a quick thread to display your production ready apps.

One rule of engagement: **Your apps must be available on some store** (App Store, Google Play, Microsoft Store...)

Post a link to your store app and some screenshots (optional).

***Mediclix - your local online pharmacy***
https://apps.apple.com/de/app/mediclix/id1632614968
https://play.google.com/store/apps/details?id=de.mediclix.app&hl=de&gl=de

Not in your region             |  Koln region free delivery support
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/34435929/186136991-713cb6cf-0de7-4e19-a469-922b8fb30215.jpg)  |  ![](https://user-images.githubusercontent.com/34435929/186136989-705ea654-9b6e-4479-8dbb-cd828f1ac836.jpg)

Search for Aspirin            |  View all products from pharmacy
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/34435929/186136985-7608a826-bb76-4e06-9014-a13841ca0e59.jpg)  |  ![](https://user-images.githubusercontent.com/34435929/186136987-5189ce73-02b1-490a-9c59-041e8f57253b.jpg)

Medicament in detail             |  Cart and checkout
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/34435929/186136981-d24d9d9d-6c13-45fb-814f-9b002baae3f8.jpg)  |  ![](https://user-images.githubusercontent.com/34435929/186136983-dc007cac-7f3b-4a8c-a0f2-fe7d9bd215db.jpg)

**Comment by RobTF** [comment]
Vismo Monitor
https://play.google.com/store/apps/details?id=com.vismo.Monitor&pli=1
https://apps.apple.com/gb/app/vismo-monitor/id1594334045

![image](https://github.com/dotnet/maui/assets/4292237/2e4c1d6f-741c-4efc-9a9a-a02b2f6233bc)
![image](https://github.com/dotnet/maui/assets/4292237/141ec721-fab3-417a-9583-5acdb552e8f5)
